examples/cors/backend/front-envoy.yaml

```diff
@@ -8,13 +8,13 @@ static_resources:
     - filters:
       - name: envoy.filters.network.http_connection_manager
         typed_config:
-          "@type": type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
```

Comment:
It would be better if this change is a separate PR but not a crisis.

Reply:
OK. Created #14125, which to be merged to release/v1.16 first and I will create a new PR for UDP fix.

Reply:
Created yet another PR #14127 for @antoniovicente fix to auth coverage test. I am going to close this PR, wait until the other two are merged to release/v1.16 and create a new PR just for UDP fixes.